README.md

```diff
@@ -659,8 +659,7 @@ Cloud Run. (Upon customizing, `PORT` value will have the specified value.)
 
 ### What's the maximum request execution time limit?
 
-Currently, a request times out after **15 minutes**. See [limits][lim].
-(60-minute request timeouts are currently in preview.)
+Currently, a request times out after **5 minutes** but can be configured up to 60 minutes. See [limits][lim].
```

Comment:
Can we say: by default 5 minutes or up to 60 minutes, if configured

Reply:
Sure, updated to:

By default 5 minutes or up to 60 minutes, if configured. See [limits][lim].